hypothesis/h
# Add annotation.sequence_id column (DB migration)

Testing:

* You might want to use https://github.com/hypothesis/h/pull/6205 as a convenient way to create annotations in the DB for testing

* Upgrade your dev DB:

      make db args='upgrade head'

* You should see that existing annotations in your DB have `NULL` in the `sequence_id` column: `tox -qqe dockercompose -- exec postgres psql -U postgres -c "select sequence_id from annotation;"`

* Add some more annotations: `tox -qe dev -- sh bin/hypothesis --dev createannotations --number 5`

* You should see that the new annotations have `sequence_id`'s starting from 15M: `tox -qqe dockercompose -- exec postgres psql -U postgres -c "select sequence_id from annotation;"`

* Delete the column: `make db args='downgrade -1'`
